components/webui/README.md

```diff
@@ -44,3 +58,68 @@ package:
    ```
 5. The Web UI should now be available at `http://<webui.host>:<webui.port>`
    (e.g., http://localhost:4000).
+
+# Linting
+
+We enforce code quality and consistency across our project using ESLint. You can use the following
+npm scripts defined in `package.json` to check and fix linting issues.
+
+## Preparing Your Environment for Linting
+
+Due to specific dependencies, linting this project requires Node.js version 18 or higher. Follow
+these steps to set up your environment for linting:
+1. **Switch to Node.js Version 18 or Higher**
+
+    If you have not installed Node.js 18 or higher, use nvm to install it first:
+    
+    ```shell
+    nvm install 18
+    ```
+    
+    Switch to Node.js 18:
+    
+    ```shell
+    nvm use 18
+    ```
```

Comment:
```suggestion
    ```shell
    # Install v18 if not already installed
    nvm install 18

    # Switch to node v18
    nvm use 18
    ```
```